### DISCUSSION TITLE
ESP Altherma/ Relais

### DISCUSSION THREAD

**Discussion by Alexander-Wehner** [raomin/ESPAltherma#269] [Q&A] [answered]
Hello, I have connected a relay to my Daikin to be able to switch it on and off.
Unfortunately I get an error message when I want to install the software, what is my error?
The relay is connected to an ESP 32 which in turn is connected to the Daikin Altherma, thanks for your help

![image](https://user-images.githubusercontent.com/112548044/230764997-2ce237ec-4fba-4691-9454-55c5331039be.png)

![image](https://user-images.githubusercontent.com/112548044/230765024-e152ea5f-f084-499d-b84f-70d374e5176d.png)

**Comment by raomin** [comment, marked as answer]
To switch ON and OFF that's not the SG1/2 pin that you need to set.
You need to wire a relay module to the external thermostat connections.
Refer to the schematic map of your heat pump to see where to connect _external On Off thermostat_.
